1-exercises/A-accessing-values/exercise3.js

```diff
@@ -21,12 +21,16 @@ let basketballTeam = {
 */
 
 // write code here
-
+let players = basketballTeam.topPlayers;
+players.sort();
+players.forEach(function (topPlayers) {
```

Comment:
This works as intended, but 2 small things:
- the name of the parameter `topPlayers` is misnamed. It will contain the name of a single player. So it's better names `topPlayer` or `player` or `playerName`. It's important (maybe not here, but in more complex code) to use variable and parameter names that reflect as precisely as possible what they're going to contain.
- you can use the arrow notation to make your code shorter and easier to read